clients/bank.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Dummy-account with a lot of funds in it to share them with others and make txs and
+ * make interactive voting/tx-sending from future REPL
+ *
+ * This one requires ENV setting with SECRET_KEY to have funds to send
+ *
+ * @module clients/bank
+ */
+
+'use strict';
+
+const evt  = require('lib/events');
+const tp   = require('core/transport');
+const sync = require('services/sync');
+const acc  = require('services/wallet');
+
+/**
+ * Amount to send when requested
+ * @type {Number}
+ */
+const AMOUNT = '0x64'; // 100
+
+(async function init() {
+
+    // Sync with other nodes if there are
+    if (tp.knownNodes.size > 1) {
+        await Promise.all([sync.pool(), sync.chain()]);
+    }
+
+})().then(async function startListening() {
+
+    // Start listening to network events right after
+    require('services/observer');
+
+    tp.on(evt.GIMME_COINS, listenAndSend);
+});
+
+/**
+ * Simply listen to GIMME_COINS event and send TX with 100 coins in response
+ *
+ * @listens events.GIMME_COINS
+ * @emits   events.NEW_TRANSACTION
+ *
+ * @param  {String} addr Address to send coins to
+ */
+function listenAndSend(addr) {
+    tp.send(evt.NEW_TRANSACTION, acc.tx('0x' + addr, AMOUNT));
```

Comment:
Correct me if I'm wrong, but if address is string it should already have `0x` prefix.

Reply:
It's not normalised. There should either be a single method defining an 'address' as it should be or the shortest/simplest version possible. And now and in [other pr](https://github.com/WingsDao/griffin-consensus-poc/pull/32) shortest method is this one.

Of course there's an obvious caveat with sending '0x' prefixed transactions which must be solved as well.

Reply:
I can agree with Roma, address in case of string representation must have '0x' as prefix, or it could be bytes array, not a string.